.prettierrc.ts

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  semi: false,
+  singleQuote: true,
```

Comment:
the single quote rule is on but I see changes that use double quote. Something is wrong